slicetag.sh

```diff
@@ -48,7 +48,7 @@ set -e
 command=$1
 shift || :
 
-TAG=$( git rev-list --all | wc -l ) 
+TAG=$( date -u +"%s" ) 
```

Comment:
Good point.  I think the most recent commit should address this, while still preserving the value of using a timestamp. However, what it doesn't address is the potential issue of building slice packages from forks of this repository, or even a branch of this same repository, that may have different commit histories. For example, say you want to test some idea, you make some commits in your fork and want  to build a new slice package for testing those changes. You push the package to the testing repo, and initiate an upgrade on some testing server.  Let's say the new package release number is 1458146913.  You then decide that those commits were a bad idea or they broke something, so you never merge them into the master branch of support repository, but meanwhile a couple commits were merged into master from a day or two ago, and you want to rebuild a slice package from _that_ for testing.  However, it turns out that the timestamp of the most recent commit in the master branch is, say, 1458060649, which is a smaller number than the release version of the other test package.  As I understand it, yum would not recognize the new testing package as an upgrade, since the release version is less than the currently installed one, and would therefore refuse to install it unless forced to do so, which would be a manual process.

Am I thinking about this improperly (not unlikely)?